tests/system/config/apm-server.yml.j2

```diff
@@ -67,6 +67,17 @@ output.elasticsearch:
       when.contains:
         processor.event: "sourcemap"
   {% endif %}
+
+  {% if index_name_transaction %}
+  indices:
+    - index: {{ index_name_transaction }}
+      when.contains:
+        processor.name: "transaction"
+    - index: {{ index_name_error }}
```

Comment:
Although this works with the test handling, it would be cleaner to also check `{% if index_name_error %}` is set